### ISSUE TITLE
Player Models

### ISSUE THREAD

**Issue by ManatoUzaki** [SaltyTheSaltShaker/JaySalty-PikCraft-NonServer#10]
I have followed the required process to install the custom Pikmin models, but they do not show up upon picking a Pikmin Origin I am left with my chosen MC skin.

**Comment by Battleboy96** [comment]
Yes! I was just about to say the same thing! I'm going to try something to se if if it works but it probably won't. Edit: the tests didn't work.

**Comment by kodie** [comment]
@ManatoUzaki @Battleboy96 You need the "Customizable Player Models" mod: https://www.curseforge.com/minecraft/mc-mods/custom-player-models-fabric/files/5473552

You'll need to bind the model selection screen button by going into your settings and then push that button and click "Models" and you should be able to select a Pikmin model.

**Comment by Battleboy96** [comment]
Thank you. I almost forgot about this stuff. I’ll keep it in mind.

-Battleboy96

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
